portfolio/src/main/webapp/style.css

```diff
@@ -80,30 +280,11 @@ li a {
   float: right;
 }
 
-.column {
-    float: left;
-    width: 30%;
-    padding: 5px;
-}
-.row::after{
-    content: "";
-    clear:both;
-    display: table;
-}
-
 p {
-    color: white;
+    color:#000033;
+    font-family: optica, sans-serif;
 }
```

Comment:
This overwrites what you defined earlier in this file (line 11). I think it's preferred to scope those general tags (p, h5, h6) to under a containing div if you want specific styling for each section (e.g. `.aboutme h5 { }`), so that it become clearer where the styling is applied. The default global ones like text color and font-family can go into the `body { }` clause.